__tests__/app.test.js

```diff
@@ -163,13 +164,155 @@ test("200: responds with updated votes", () => {
  })
 })
 
-describe("Path entered doesn't match an existing path", () => {
-  test("404: responds with error for a route that is not available", () => {
-  return request(app)
-    .get("/api/user")
-    .expect(404)
-    .then(({body})=> {
-      expect(body.msg).toBe("The endpoint does not exist")
+
+
+describe("8. GET: /api/articles", () => {
+  
+  test("200: responds with correct content keys", () => {
+   return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then((response) => {
+      let {
+        _body: articles
+      } = response
+      articles = articles.article
+          articles.forEach((article) => {
```

Comment:
We should be checking the length of the articles coming back before we use forEach to check all of them. If what came back was an empty, then we would never get to the assertion and this test would break our code.